pheps/phep-9999.md

```diff
@@ -0,0 +1,112 @@
+   - PyHC Env Installation Conflicts: indicates state of installation conflicts within the PyHC software environment
+   - DOI: indicates whether or not a package has a DOI (e.g., from Zenodo or a publication)
+   - Interoperability Status: indicates the level of interoperability a package has with PyHC core packages
+   - pip Installable: indicates whether a package is pip installable
+   - conda Installable: indicates whether a package is conda installable
+
+The following table shows the benefits that are associated with each tier:
+
+| Tier                  | Summer School Inclusion | PyHC Software Env Inclusion | PyHC-Chat Bot Inclusion | pyOpenSci Verified badge | Standards Compliance Assistance | Listing on Main PyHC Project Page | Listing on Secondary PyHC Project Page | Software Search Interface Inclusion | Consideration for Conference Travel Funding |
+| :--:                  | :---------------------: | :-------------------------: | :---------------------: | :----------------------: | :-----------------------------: | :-------------------------------: | :------------------------------------: | :---------------------------------: | :-----------------------------------------: |
+| **Gold**              | Yes                     | Yes                         | Yes                     | Yes                      | Yes                             | Yes                               | No                                     | Yes                                 | Yes                                         |
+| **Bronze**            | No                      | No                          | No                      | No                       | Yes                             | Yes                               | No                                     | Yes                                 | Yes                                         |
+| **Silver**            | No                      | No                          | No                      | No                       | No                              | Yes                               | No                                     | Yes                                 | No                                          |
+| **Honorable Mention** | No                      | No                          | No                      | No                       | No                              | No                                | Yes                                    | Yes                                 | No                                          |
+
+Descriptions for each heading are as follows:
+   - Summer School Inclusion: indicates whether a package will be included in summer school teaching materials
+   - PyHC Software Env Inclusion: indicates whether a package will be included within the PyHC software environment
+   - PyHC-Chat Bot Inclusion: indicates whether a packages will have up-to-date information included within the ChatGPT4-powered PyHC-Chat bot
```

Comment:
Recommend having these be something people can opt in or out of.

Reply:
Didn't even think of that, that's a good idea. I'll modify to indicate that that would be an optional perk.

Reply:
I would recommend dropping this from the table completely. I am sure there will be lots of smaller things like this that projects will or wont get pulled into. This feels very frivolous to include in a very formal specification document.

Reply:
@Cadair I disagree that it's frivolous. As PyHC-Chat improves, inclusion within it could be a carrot—not the only, or biggest, but still a carrot—for people to work to move tiers.